firebase-functions/functions/src/models/player-choice.ts

```diff
@@ -0,0 +1,8 @@
+﻿import { Choice } from './choice';
+import { Result } from './result';
+
+export interface PlayerChoice {
+  playerId: string;
+  choice: Choice;
+  result: Result;
```

Comment:
Maybe the description of my intention in the PR was a little fuzzy. I save this here so it can later be used for easy 
maningful aggregation of the statistics, e.g:

- When showing the current rooms statistics, color winning symbols green and losing ones red in each round
- Globally, show how often a player (or anyone) has won using a specific symbol (so has stone a higher chance of winning globally? :))

We might give the Result something similar though, but only with something like `resolved` or `draw`. That could answer the question of how often a round gets resolved for a specific number of players or something, but I am not sure if that is that interesting. Thoughts and comments on my reasoning here please?